x/epochs/keeper/epoch_test.go

```diff
@@ -6,6 +6,60 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 )
 
+func (suite *KeeperTestSuite) TestAddEpochInfo() {
+	defaultIdentifier := "default_add_epoch_info_id"
```

Comment:
These might better be consts since these never change

Reply:
eh, I'm not sure I really buy into the notion that const vs not const matters for variables at the top of a test. If it was file-wide, I'd totally agree